playbooks/ha_log_cfg_sync.yaml

```diff
@@ -65,6 +65,13 @@
         content: "{{ rsync_password }}"
         mode: 0600
 
+    - name: Teardown original slave rsyncd(when slave master switched)
```

Comment:
Just a question: I can't find any changes you mentioned in comment msg: We also need to remove the related crontab on ori master nodes, which are already done in ha_log_cfg_sync playbook?

Reply:
Yeah, the part you mentioned already done in that file. I will udpate the commit mesg